app/grandchallenge/reader_studies/forms.py

```diff
@@ -120,6 +121,33 @@
 }
 
 
+def verify_cleaned_html(
```

Comment:
This is a little bit unique when it doesn't need to be. It would be better implemented as a validator, then it will fit in with the rest of the ecosystem: https://docs.djangoproject.com/en/5.2/ref/validators/